### ISSUE TITLE
“xcodebuild” command but it exited with error code 65

### ISSUE THREAD

**Issue by ldco2016** [facebook/react-native#25240]
After upgrading to React Native 0.59.9 and React 16.8.3 and attempted to run `react-native run-ios`, I get this error:

```
info Undefined symbols for architecture x86_64:
  "_JSClassCreate", referenced from:
      facebook::jsc::JSCRuntime::createObject(std::__1::shared_ptr<facebook::jsi::HostObject>)::$_0::operator()() const in libReact.a(JSCRuntime.o)
      facebook::jsc::JSCRuntime::createFunctionFromHostFunction(facebook::jsi::PropNameID const&, unsigned int, std::__1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>)::$_1::operator()() const in libReact.a(JSCRuntime.o)
  "_JSContextGetGlobalObject", referenced from:
      facebook::jsc::JSCRuntime::global() in libReact.a(JSCRuntime.o)
      facebook::jsc::JSCRuntime::createFunctionFromHostFunction(facebook::jsi::PropNameID const&, unsigned int, std::__1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>)::HostFunctionMetadata::initialize(OpaqueJSContext const*, OpaqueJSValue*) in libReact.a(JSCRuntime.o)
  "_JSEvaluateScript", referenced from:
      facebook::jsc::JSCRuntime::evaluateJavaScript(std::__1::unique_ptr<facebook::jsi::Buffer const, std::__1::default_delete<facebook::jsi::Buffer const> >, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in libReact.a(JSCRuntime.o)
  "_JSGlobalContextCreateInGroup", referenced from:
      facebook::jsc::JSCRuntime::JSCRuntime() in libReact.a(JSCRuntime.o)
      facebook::jsc::JSCRuntime::JSCRuntime() in libReact.a(JSCRuntime.o)
  "_JSGlobalContextRelease", referenced from:
      facebook::jsc::JSCRuntime::JSCRuntime() in libReact.a(JSCRuntime.o)
      facebook::jsc::JSCRuntime::~JSCRuntime() in libReact.a(JSCRuntime.o)
      facebook::jsc::JSCRuntime::JSCRuntime() in libReact.a(JSCRuntime.o)
  "_JSGlobalContextRetain", referenced from:
      facebook::jsc::JSCRuntime::JSCRuntime(OpaqueJSContext*) in libReact.a(JSCRuntime.o)
  "_JSObjectCallAsConstructor", referenced from:
      facebook::jsc::JSCRuntime::callAsConstructor(facebook::jsi::Function const&, facebook::jsi::Value const*, unsigned long) in libReact.a(JSCRuntime.o)
  "_JSObjectCallAsFunction", referenced from:
      facebook::jsc::JSCRuntime::call(facebook::jsi::Function const&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long) in libReact.a(JSCRuntime.o)
  "_JSObjectCopyPropertyNames", referenced from:
      facebook::jsc::JSCRuntime::getPropertyNames(facebook::jsi::Object const&) in libReact.a(JSCRuntime.o)

error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening NFIBEngage.xcworkspace

** BUILD FAILED **


The following commands produced analyzer issues:
	Analyze /Users/danale/Projects/engage-application.mobile/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp normal x86_64
	Analyze Base/RCTModuleMethod.mm normal x86_64
(2 commands with analyzer issues)
```

React Native version:

```
info
  React Native Environment Info:
    System:
      OS: macOS High Sierra 10.13.6
      CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
      Memory: 161.54 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 11.10.1 - /usr/local/bin/node
      Yarn: 1.10.1 - /usr/local/bin/yarn
      npm: 6.7.0 - /usr/local/bin/npm
      Watchman: 4.7.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        API Levels: 23, 25, 26, 27, 28
        Build Tools: 23.0.1, 26.0.2, 27.0.3, 28.0.3
    IDEs:
      Android Studio: 3.4 AI-183.5429.30.34.5452501
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3
      react-native: 0.59.9 => 0.59.9
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7
```

## Steps To Reproduce

1. Run `react-native run-ios`

Describe what you expected to happen:

Well, since I did upgrade I had expected some bugs, but I am unfamiliar with the inner workings of Xcode in conjunction with React Native.

Snack, code example, or link to a repository:

https://www.dropbox.com/s/ygyrhnno8aou4jg/RNsampleApp.zip?dl=0

I was eventually able to resolve the above error by following this guide:

https://stackoverflow.com/questions/54515175/undefined-symbols-for-architecture-arm64-jsclasscreate

and running `react-native link`.

However, now inside of Metro Bundler I get this error:

```
VM16:1 GET http://localhost:8081/Users/danale/Library/Developer/CoreSimulator/Devices/9E045643-4D13-41F4-A144-781A6513E884/data/Containers/Bundle/Application/AC51BEE9-3B17-4246-A747-D9D47B460B58/NFIBEngage.app/main.jsbundle 404 (Not Found)
(anonymous) @ VM16:1
deltaUrlToBlobUrl @ deltaUrlToBlobUrl.js:31
getBlobUrl @ (index):209
ws.onmessage @ (index):184
(index):190 Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0
```

I am not exactly sure what is going on there, but its referencing a line of code found inside of `ios/NFIBEngage.xcodeproj/project.pbxproj`:

```
/* Begin PBXFileReference section */
		008F07F21AC5B25A0029DE68 /* main.jsbundle */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = main.jsbundle; sourceTree = "<group>"; };
```

```
13B07FAE1A68108700A75B9A /* NFIBEngage */ = {
			isa = PBXGroup;
			children = (
				6B2F5E9F2007D4B500FF4B14 /* UA.entitlements */,
				6B2F5E9E2007D4AC00FF4B14 /* QA.entitlements */,
				6B2F5E9D2007D49F00FF4B14 /* DEV.entitlements */,
				6B2F5E9C2007D44300FF4B14 /* PROD.entitlements */,
				6BC6C0D81FD1CB6400AD9FC6 /* Configuration */,
				008F07F21AC5B25A0029DE68 /* main.jsbundle */,

```
When I attempt a `react-native run-ios` a second time, I still get a successful build in Xcode but with this error:

```
info ** BUILD SUCCEEDED **


info Installing build/NFIBEngage/Build/Products/Debug-iphonesimulator/NFIBEngage.app
An error was encountered processing the command (domain=com.apple.CoreSimulator.SimError, code=164):
Unable to lookup in current state: Shutdown
Print: Entry, ":CFBundleIdentifier", Does Not Exist
error Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/NFIBEngage/Build/Products/Debug-iphonesimulator/NFIBEngage.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist
```

and when I attempt to open the app in the Simulator, it does not boot up and in debugger-ui I get this error:

```
VM16:1 GET http://localhost:8081/Users/danale/Library/Developer/CoreSimulator/Devices/9E045643-4D13-41F4-A144-781A6513E884/data/Containers/Bundle/Application/AC51BEE9-3B17-4246-A747-D9D47B460B58/NFIBEngage.app/main.jsbundle 404 (Not Found)
(anonymous) @ VM16:1
deltaUrlToBlobUrl @ deltaUrlToBlobUrl.js:31
getBlobUrl @ (index):209
ws.onmessage @ (index):184
(index):190 Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0
async function (async)
ws.onmessage @ (index):184
18(index):115 WebSocket connection to 'ws://localhost:8081/debugger-proxy?role=debugger&name=Chrome' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
```

It gets to the splash screen with the above error and immediately crashes. I then tried adding this script:

`"build:ios": "react-native bundle --entry-file='index.ios.js' --bundle-output='./ios/NFIBEngage/main.jsbundle' --dev=false --platform='ios' --assets-dest='./ios'"`, but instead of fixing the issue I got this error:

```
npm run build:ios

> NFIBEngage@1.0.7 build:ios /Users/danale/Projects/engage-application.mobile
> react-native bundle --entry-file='index.ios.js' --bundle-output='./ios/NFIBEngage/main.jsbundle' --dev=false --platform='ios' --assets-dest='./ios'

Loading dependency graph, done.
transform[stderr]: Trace: The node type SpreadProperty has been renamed to SpreadElement
transform[stderr]:     at Object.isSpreadProperty (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/types/lib/validators/generated/index.js:4304:11)
transform[stderr]:     at hasSpread (/Users/danale/Projects/engage-application.mobile/node_modules/babel-plugin-transform-object-rest-spread/lib/index.js:38:13)
transform[stderr]:     at PluginPass.ObjectExpression (/Users/danale/Projects/engage-application.mobile/node_modules/babel-plugin-transform-object-rest-spread/lib/index.js:234:14)
transform[stderr]:     at newFn (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/traverse/lib/visitors.js:193:21)
transform[stderr]:     at NodePath._call (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/traverse/lib/path/context.js:53:20)
transform[stderr]:     at NodePath.call (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/traverse/lib/path/context.js:40:17)
transform[stderr]:     at NodePath.visit (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/traverse/lib/path/context.js:88:12)
transform[stderr]:     at TraversalContext.visitQueue (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/traverse/lib/context.js:118:16)
transform[stderr]:     at TraversalContext.visitSingle (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/traverse/lib/context.js:90:19)
transform[stderr]:     at TraversalContext.visit (/Users/danale/Projects/engage-application.mobile/node_modules/@babel/traverse/lib/context.js:146:19)
error node_modules/react-native-keyboard-manager/index.js: Cannot read property 'bindings' of null. Run CLI with --verbose flag for more details.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! NFIBEngage@1.0.7 build:ios: `react-native bundle --entry-file='index.ios.js' --bundle-output='./ios/NFIBEngage/main.jsbundle' --dev=false --platform='ios' --assets-dest='./ios'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the NFIBEngage@1.0.7 build:ios script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/danale/.npm/_logs/2019-06-13T17_58_48_335Z-debug.log
```

Then I tried booting up the application via Xcode and it boots up the simulator and attempts to open the application inside of simulator but it hangs on splash screen and Xcode gave me this error:

```
error: EOF while parsing a value at line 1 column 0

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
Please attach the full debug log to all bug reports.
```

and it also indicates this configuration inside of Xcode:

```
/**
 * SplashScreen
 * 启动屏
 * from：http://www.devio.org
 * Author:CrazyCodeBoy
 * GitHub:https://github.com/crazycodeboy
 * Email:crazycodeboy@gmail.com
 */

#import "SplashScreen.h"
#import <React/RCTBridge.h>

static bool waiting = true;
static bool addedJsLoadErrorObserver = false;

@implementation SplashScreen
- (dispatch_queue_t)methodQueue{
    return dispatch_get_main_queue();
}
RCT_EXPORT_MODULE()

+ (void)show {
    if (!addedJsLoadErrorObserver) {
        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(jsLoadError:) name:RCTJavaScriptDidFailToLoadNotification object:nil];
        addedJsLoadErrorObserver = true;
    }

    while (waiting) {
        NSDate* later = [NSDate dateWithTimeIntervalSinceNow:0.1];
        [[NSRunLoop mainRunLoop] runUntilDate:later]; Thread 1: signal SIGBART
    }
}
```

I am still stuck on this `main.jsbundle` not being found. Can anyone assist here?

**Comment by KingAmo** [comment]
i have the same error when i use `react-native run-ios` ， but in Xcode，click the `run` button ，everything is fine

**Comment by AlbertVilaCalvo** [comment]
I just got this error after upgrading to 0.60.4 (from 0.59.2).

To fix it I've had to rename the xcworkspace folder from gazella.xcworkspace to Gazella.xcworkspace.

I could successfully run the project from within XCode, but got the error when doing react-native run-ios on the terminal. Now everything works fine.

**Comment by dilipchandima** [comment]
I have the same error after upgrading to 0.60.4 from 0.59.9

@AlbertVilaCalvo your solution is not working for me

**Comment by dilipchandima** [comment]
```
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening mobile.xcworkspace. Run CLI with --verbose flag for more details.
User defaults from command line:
    IDEDerivedDataPathOverride = /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile

Prepare build
note: Using legacy build system

=== BUILD AGGREGATE TARGET AppCenter OF PROJECT Pods WITH CONFIGURATION Debug ===

Check dependencies

=== BUILD AGGREGATE TARGET AppCenterReactNativeShared OF PROJECT Pods WITH CONFIGURATION Debug ===

Check dependencies

=== BUILD AGGREGATE TARGET Fabric OF PROJECT Pods WITH CONFIGURATION Debug ===

Check dependencies

=== BUILD AGGREGATE TARGET Crashlytics OF PROJECT Pods WITH CONFIGURATION Debug ===

Check dependencies

=== BUILD TARGET DoubleConversion OF PROJECT Pods WITH CONFIGURATION Debug ===

Check dependencies

Write auxiliary files
write-file /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-all-non-framework-target-headers.hmap
write-file /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-all-target-headers.hmap
write-file /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-project-headers.hmap
write-file /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-own-target-headers.hmap
write-file /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion.hmap

CompileC /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/cached-powers.o DoubleConversion/double-conversion/cached-powers.cc normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
    cd /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods
    export LANG=en_US.US-ASCII
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/dileepa/programs/jdk/Contents/Home//bin:/Users/dileepa/Library/Android/sdk/tools:/Users/dileepa/Library/Android/sdk/platform-tools:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c++ -arch x86_64 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu++14 -stdlib=libc++ -fmodules -fmodules-cache-path=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Werror=deprecated-objc-isa-usage -Werror=objc-root-class -Wno-non-virtual-dtor -Wno-overloaded-virtual -Wno-exit-time-destructors -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wno-newline-eof -Wno-c++11-extensions -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.2.sdk -fasm-blocks -fstrict-aliasing -Wdeprecated-declarations -Winvalid-offsetof -mios-simulator-version-min=9.0 -g -Wno-sign-conversion -Winfinite-recursion -Wmove -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wrange-loop-analysis -Wno-semicolon-before-method-body -Wunguarded-availability -index-store-path /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Index/DataStore -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-generated-files.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-own-target-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-all-target-headers.hmap -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-project-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion/include -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources-normal/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources -F/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion -Wno-unreachable-code -include /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Target\ Support\ Files/DoubleConversion/DoubleConversion-prefix.pch -MMD -MT dependencies -MF /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/cached-powers.d --serialize-diagnostics /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/cached-powers.dia -c /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/DoubleConversion/double-conversion/cached-powers.cc -o /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/cached-powers.o

CompileC /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/DoubleConversion-dummy.o Target\ Support\ Files/DoubleConversion/DoubleConversion-dummy.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    cd /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods
    export LANG=en_US.US-ASCII
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/dileepa/programs/jdk/Contents/Home//bin:/Users/dileepa/Library/Android/sdk/tools:/Users/dileepa/Library/Android/sdk/platform-tools:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -arch x86_64 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu11 -fobjc-arc -fobjc-weak -fmodules -fmodules-cache-path=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Wno-implicit-atomic-properties -Werror=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror=objc-root-class -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.2.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -mios-simulator-version-min=9.0 -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -Wunguarded-availability -fobjc-abi-version=2 -fobjc-legacy-dispatch -index-store-path /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Index/DataStore -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-generated-files.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-own-target-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-all-target-headers.hmap -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-project-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion/include -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources-normal/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources -F/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion -include /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Target\ Support\ Files/DoubleConversion/DoubleConversion-prefix.pch -MMD -MT dependencies -MF /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/DoubleConversion-dummy.d --serialize-diagnostics /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/DoubleConversion-dummy.dia -c /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Target\ Support\ Files/DoubleConversion/DoubleConversion-dummy.m -o /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/DoubleConversion-dummy.o
error: PCH was compiled with module cache path '/Users/dileepa/projects/rhino/comapny/androidX/mobile/ios/build/mobile/ModuleCache.noindex/3DNI3SU7WPYK3', but the path is currently '/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex/3DNI3SU7WPYK3'
1 error generated.

CompileC /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/bignum-dtoa.o DoubleConversion/double-conversion/bignum-dtoa.cc normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
    cd /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods
    export LANG=en_US.US-ASCII
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/dileepa/programs/jdk/Contents/Home//bin:/Users/dileepa/Library/Android/sdk/tools:/Users/dileepa/Library/Android/sdk/platform-tools:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c++ -arch x86_64 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu++14 -stdlib=libc++ -fmodules -fmodules-cache-path=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Werror=deprecated-objc-isa-usage -Werror=objc-root-class -Wno-non-virtual-dtor -Wno-overloaded-virtual -Wno-exit-time-destructors -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wno-newline-eof -Wno-c++11-extensions -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.2.sdk -fasm-blocks -fstrict-aliasing -Wdeprecated-declarations -Winvalid-offsetof -mios-simulator-version-min=9.0 -g -Wno-sign-conversion -Winfinite-recursion -Wmove -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wrange-loop-analysis -Wno-semicolon-before-method-body -Wunguarded-availability -index-store-path /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Index/DataStore -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-generated-files.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-own-target-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-all-target-headers.hmap -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-project-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion/include -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources-normal/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources -F/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion -Wno-unreachable-code -include /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Target\ Support\ Files/DoubleConversion/DoubleConversion-prefix.pch -MMD -MT dependencies -MF /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/bignum-dtoa.d --serialize-diagnostics /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/bignum-dtoa.dia -c /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/DoubleConversion/double-conversion/bignum-dtoa.cc -o /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/bignum-dtoa.o

CompileC /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/double-conversion.o DoubleConversion/double-conversion/double-conversion.cc normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
    cd /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods
    export LANG=en_US.US-ASCII
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/dileepa/programs/jdk/Contents/Home//bin:/Users/dileepa/Library/Android/sdk/tools:/Users/dileepa/Library/Android/sdk/platform-tools:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c++ -arch x86_64 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu++14 -stdlib=libc++ -fmodules -fmodules-cache-path=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Werror=deprecated-objc-isa-usage -Werror=objc-root-class -Wno-non-virtual-dtor -Wno-overloaded-virtual -Wno-exit-time-destructors -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wno-newline-eof -Wno-c++11-extensions -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.2.sdk -fasm-blocks -fstrict-aliasing -Wdeprecated-declarations -Winvalid-offsetof -mios-simulator-version-min=9.0 -g -Wno-sign-conversion -Winfinite-recursion -Wmove -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wrange-loop-analysis -Wno-semicolon-before-method-body -Wunguarded-availability -index-store-path /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Index/DataStore -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-generated-files.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-own-target-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-all-target-headers.hmap -iquote /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DoubleConversion-project-headers.hmap -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion/include -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Private/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Headers/Public/DoubleConversion -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources-normal/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources/x86_64 -I/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/DerivedSources -F/Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Products/Debug-iphonesimulator/DoubleConversion -Wno-unreachable-code -include /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/Target\ Support\ Files/DoubleConversion/DoubleConversion-prefix.pch -MMD -MT dependencies -MF /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/double-conversion.d --serialize-diagnostics /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/double-conversion.dia -c /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/Pods/DoubleConversion/double-conversion/double-conversion.cc -o /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/double-conversion.o


** BUILD FAILED **


The following build commands failed:
	CompileC /Users/dileepa/projects/rhino/comapny/comapny-mobile-client/ios/build/mobile/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/DoubleConversion.build/Objects-normal/x86_64/DoubleConversion-dummy.o Target\ Support\ Files/DoubleConversion/DoubleConversion-dummy.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

This is my console

**Comment by tusharmutreja** [comment]
Hey, I have the same error when I run **react-native run-ios** after upgrading the react-native version to 0.59.5.

And also Xcode not able to open the project.

Any Help?

**Comment by poojapmagar** [comment]
Facing same error is there any solution for this

**Comment by AdeelMughal** [comment]
Same issue. Any solution so far ?

**Comment by nicolas-rohricht** [comment]
Hi guys.
Did you find any solution for that?

**Comment by marciosalinodias** [comment]
Hey guys,
Same error.

I fixed just by changing the name from "AwesomeProject" to "awesomeproject"

Works for me  ;)

**Comment by git-sid-go** [comment]
Hello! I faced this issue with React Native version `0.61.2` and this worked for me.
Navigate to the `ios` folder of the app and run the command `pod install`
Once that is done, go back to the root directory and run `react-native run-ios`

**Comment by vibdheeraj** [comment]
@git-sid-go thanx this solution work for me.

**Comment by azizuysal** [comment]
In your project folder:
```
cd ios
rm -rf build/
cd ..
```
Run react-native run-ios again.

**Comment by josemvcerqueira** [comment]
Make sure you open your /ios on xcode and go to workspace settings and set the build system to legacy

**Comment by JoshK2** [comment]
> Hello! I faced this issue with React Native version `0.61.2` and this worked for me.
> Navigate to the `ios` folder of the app and run the command `pod install`
> Once that is done, go back to the root directory and run `react-native run-ios`

It's worked for me, thanks!

**Comment by winni4eva** [comment]
> Hello! I faced this issue with React Native version `0.61.2` and this worked for me.
> Navigate to the `ios` folder of the app and run the command `pod install`
> Once that is done, go back to the root directory and run `react-native run-ios`

This worked for me

**Comment by vaekek** [comment]
I am facing this problem with react native 0.59.10 and expo, is there any solution?

**Comment by iampeterbanjo** [comment]
i had the same issue and restarting my computer worked 🤷‍♂

**Comment by Ashutosh8127** [comment]
go inside ios folder in terminal
cd ios/
run pod install
after run react-native run-ios

**Comment by RupamShaw** [comment]
> I just got this error after upgrading to 0.60.4 (from 0.59.2).
> 
> To fix it I've had to rename the xcworkspace folder from gazella.xcworkspace to Gazella.xcworkspace.
> 
> I could successfully run the project from within XCode, but got the error when doing react-native run-ios on the terminal. Now everything works fine.
worked for me  by changing Uppercase both files .xcodeproj and .xcworkspace

**Comment by mvfsillva** [comment]
To solve this problem, I did something like @Ashutosh8127:

```sh
cd ios/
$ pod install
cd ..
$ npm run ios #or npx react-native run-ios
```

**Comment by shixiaoquan** [comment]
> In your project folder:
> 
> ```
> cd ios
> rm -rf build/
> cd ..
> ```
> 
> Run react-native run-ios again.

It's worked for me tks

**Comment by diegotauchert** [comment]
> I resolved my issue changing the Build Active Architecture Only to:
> 
> Debug >> Yes
> Release >> No
> 
> On xCode, select your Project Target >> Build Settings >> Architetures >> Build Active Architecture Only.
> 
> After changing that, react-native run-ios was executed like a charm.

it worked for me... thanks 👍🏼

**Comment by engtom** [comment]
This may occur especially when you change the bundle ID, the provisioning profiles, create new targets etc. So only running pod install may not work. In this cases, we have to also remove the build folder, like @azizuysal said, executing this:

```
- cd ios
- rm -rf build/

```
After this everything should get back to normality.

**Comment by chacaa** [comment]
Try changing the `Build System` property from `New Build System (Default)` to `Legacy Build System`. You can do it from here `File -> Workspace Settings`. That change did the job for me, hope it helps you!

**Comment by kockok** [comment]
> ```shell
> pod install
> ```

I got `error code 65` after updating to the latest react-native,
the above steps worked.

**Comment by ayepRahman** [comment]
> Hello! I faced this issue with React Native version `0.61.2` and this worked for me.
> Navigate to the `ios` folder of the app and run the command `pod install`
> Once that is done, go back to the root directory and run `react-native run-ios`

Work! great!

**Comment by danielkiesshau** [comment]
I had a "react-native" module installed globally. After I uninstalled it, I could start my project

**Comment by hsh2001** [comment]
I got the same error when I tried the way in the comments above. But when I reinstalled Cocoapods, I solved the problem! Try it.

Step 1. 
```bash
cd ios;
rm -rf Pods;
rm -rf build; # may need sudo

brew uninstall cocoapods;
brew install cocoapods;

pod install;
cd ..;
```

**Comment by sjc5** [comment]
> This may occur especially when you change the bundle ID, the provisioning profiles, create new targets etc. So only running pod install may not work. In this cases, we have to also remove the build folder, like @azizuysal said, executing this:
> 
> ```
> - cd ios
> - rm -rf build/
> ```
> 
> After this everything should get back to normality.

This worked for me after also executing "pod install" in the ios folder. Not sure whether that step is prerequisite but thought I'd mention.

**Comment by Khushboo-gupta-123** [comment]
when i run my project on  terminal i faced this issuue. is there any solution?
"Failed to build iOS project. We ran "xcodebuild" command but it exited with error code null. To debug build logs further, consider building your app with Xcode.app, by opening demo.xcworkspace "

**Comment by 234146326** [comment]
My error ! 
`error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening react_native_test1.xcworkspace. Run CLI with --verbose flag for more details.`
My solution：
The “Metro Bundler terminal” running。After closing, run react-native run-ios. Success.

》Hope that can help！

**Comment by lucasR23** [comment]
I got the same error and in the folder of the project i had to run:
```
- yarn
- cd ios
- pod install
```

**Comment by GuilhermeTrivilin** [comment]
What worked for me was to rename the project by removing the special characters.

Example: "project_marketplace" to "projectmarketplace"

In this case, I redid the project with react-native init and copied the src and package.json folder.

**Comment by knudnilsen** [comment]
I had another progress running on port 8081. Just stopped that service and - et voila - works again

**Comment by guistrutzki** [comment]
> i had the same issue and restarting my computer worked 🤷‍♂

I tried all the other solutions, but just this one worked. So ridiculous but thanks haha =D

**Comment by evelew** [comment]
I tried all the solutions, and after all I realized that I was trying to run the app at one port that was running an web application, after stop the web application the react native app runs normally

**Comment by shixiaoquan** [comment]
> > i had the same issue and restarting my computer worked man_shrugging
> 
> I tried all the other solutions, but just this one worked. So ridiculous but thanks haha =D

sometimes,restart is the best solution,

**Comment by joeponzio** [comment]
I had this issue and it was because I had renamed my XCode targets to `Development` and `Production` (and hence build schemes in XCode) which don't match my workspace/project name. `npx react-native run-ios` was looking for a target matching my workspace name, but that no longer existed. I solved this by specifying which scheme to use, which is not a hack or workaround but a reasonable parameter to add if you have multiple schemes:

`npx react-native run-ios --scheme "Development"`

After a minute of XCode rebuilding my project (through the command line, so you don't see a ton of progress), I am now up and running.

**Comment by markuskoehler360** [comment]
after reading the topmost answer in this thread, tried running it with xcode. there i could debug my errors. in my case i was running another builder on port 8081. the port react-native wants to use. 
after killing this process, i was able to build my app using 'react-native run-ios'

i hope this helps anyone!

**Comment by markuskoehler360** [comment]
> > > i had the same issue and restarting my computer worked man_shrugging
> > 
> > 
> > I tried all the other solutions, but just this one worked. So ridiculous but thanks haha =D
> 
> sometimes,restart is the best solution,

maybe you had the same issue as me, and restarting led to a free 8081 port!

**Comment by khanof89** [comment]
I was getting this error because I ran react-native link, removing the link 
`react-native unlink react-native-vector-icons` 
and then running `react-native run-ios` and everything worked fine

**Comment by tksilicon** [comment]
> Hello! I faced this issue with React Native version `0.61.2` and this worked for me.
> Navigate to the `ios` folder of the app and run the command `pod install`
> Once that is done, go back to the root directory and run `react-native run-ios`

This helped. I do not know why they are downvoting it.

**Comment by sumanthyedoti** [comment]
> I was getting this error because I ran react-native link, removing the link
> `react-native unlink react-native-vector-icons`
> and then running `react-native run-ios` and everything worked fine

The true reason 😄

**Comment by wemboava** [comment]
go to an ios folder on the terminal
$ `cd ios`
install the pod
$ `pod install`
if you don't have cocoapods installed, run
$ `brew install cocoapods`
go back a folder
$ `cd ..`
and finally
$ `react-native run-ios`

**Comment by jeroen-van-dijk** [comment]
Make sure port 8081 is not in use

**Comment by Liqiankun** [comment]
* `rm -rf node_modules`
* `rm -rf build`
* `yarn`
* `npx react-native run-ios`

This works for me.

**Comment by GardEngen** [comment]
cd ios/
pod install

**Comment by leopavez** [comment]
Hi guys, same problem from here.
I fixed it by running `react-native unlink react-native-vector-icons`, for some reason it doesn't work with that installation method

**Comment by wesson818** [comment]
I got the same issue. I tried lots of different ways listed above. Didn't work for me.
I fixed by remove all .ttf lines in /ios/exampleApp.xcodeproj/project.pbxproj which listed in XCode when I build. 
Hope it helps if other solutions does work.

**Comment by safaiyeh** [comment]
Thanks for all the solutions everyone! The original issue seems to be a bit dated. Closing, please open a new issue if this occurs in the latest RN version and none of these solutions work.

**Comment by Preetha-stack** [comment]
Hi guys, Same problem..tried with all solutions above mentioned!!

**Comment by littlepriest** [comment]
> Hello! I faced this issue with React Native version `0.61.2` and this worked for me.
> Navigate to the `ios` folder of the app and run the command `pod install`
> Once that is done, go back to the root directory and run `react-native run-ios`

thanks. it really works for me ! so nice!!!!

**Comment by Lisba** [comment]
> Hello! I faced this issue with React Native version `0.61.2` and this worked for me.
> Navigate to the `ios` folder of the app and run the command `pod install`
> Once that is done, go back to the root directory and run `react-native run-ios`

Thanks!!

**Comment by init-JaeminKim** [comment]
In my case,

1. Clone repo
2. npm install
3. cd ios && pod install && cd ..

**Comment by tomfriwel** [comment]
> I was getting this error because I ran react-native link, removing the link
> `react-native unlink react-native-vector-icons`
> and then running `react-native run-ios` and everything worked fine

I use both `react-native unlink react-native-vector-icons` and `cd ios && rm -rf build/`.

**Comment by init-JaeminKim** [comment]
1. The only way I took was cloning repo again by the time error codes popped up
2. Go to react-native-vector-icons GitHub
3. Follow the **Installation** steps.
  - I used **iOS** Option: Manually
4. Don't forget **pod install** in your project's root directory run `cd ios && pod install && cd ..`
5. react-native run-ios

**Comment by pea-nut-z** [comment]
This fixed my problem:
https://github.com/facebook/react-native/issues/30836
Try checking your error log on xcode, it gives more details.

**Comment by AmirDiafi** [comment]
The only solution I found was to create a new RN project but with version 60. Example:

react-native init myNewApp --version 0.60.0

**Comment by Apatoallo** [comment]
> i had the same issue and restarting my computer worked 🤷‍♂

Hahahah

**Comment by Wado317** [comment]
Go to ios/Podfile comment out Flipper, then in the console: cd ios, pod install, cd .. then start metro server and run your ios project. Should be working now :)

**Comment by rldona** [comment]
I share with you the solution that has worked for me...

It seems that the behavior of a project in **iOS** is very similar to a project in **JavaScript**. Sometimes you have to delete the _node_modules_ folder and reinstall the dependencies.

**Follow the next steps:**

Close Metro in the terminal and the app in the iOS simulator (remove the app from the simulator if necessary)

```Terminal
cd ios
```

```Terminal
rm -rf Pods
```

```Terminal
rm -rf Podfile.lock
```

```Terminal
pod install
```

You can finally rebuild the project:

```Terminal
npx react-native run-ios
```

Good luck and I hope it helps 🚀

**Comment by Aditya23456** [comment]
> Go to ios/Podfile comment out Flipper, then in the console: cd ios, pod install, cd .. then start metro server and run your ios project. Should be working now :)

Thank you very much. I tried each and every fix mentioned in the thread but nothing worked. Atlast, removing flipper and it was fixed. Is this a desirable fix or will removing flipper cause any issues?

**Comment by Khubaib-Sarfraz** [comment]
solution is here 
just delete your podFile of your project and run command 
pod init 
inside the ios directory of your project 
this will slove the issue

**Comment by tanloiit2010** [comment]
> i had the same issue and restarting my computer worked 🤷‍♂

This work for me. Thank you!!!

**Comment by abdul-ahad-devs** [comment]
I have tried all of the above mentioned solutions but none of them worked for me. I have been using xcode version 12.5.1, react-native version 0.64 and my cocoapods version is 1.10.2

**Comment by fouss6C** [comment]
Guy , don't waste your time. 
This issue may depends on your environnement , I did everything here.
But what fixed mine is just to comment use_flipper() line on podfile under iOS dir.

**Comment by rajiv-eulogik** [comment]
Damn these answers are more confusing

**Comment by san-Gold** [comment]
My project is solved in this way：
yarn add react-native-reanimated@^2.3.0

**Comment by ItztrippleA** [comment]
> In your project folder:
> 
> ```
> cd ios
> rm -rf build/
> cd ..
> ```
> 
> Run react-native run-ios again.

did not work  for me

**Comment by MartinDKnudsen** [comment]
dont do cd iOS

Just be in your root folder and run 
`npx pod-install`

:-)

**Comment by okanthony** [comment]
Looks as though there is a known issue with Flipper and `react-native@0.66`. I'm also using `nvm` and saw some errors around incorrect prefix, so I:
- Commented out `use_flipper!()` in `ios/Podfile`
- Deleted `ios/Pods` and `Podfile.lock`
- In `ios` dir ran `pod install`
- Completely uninstalled `node` and `nvm`
- Reinstalled `nvm`, `node`, and `npm`

I was then able to successfully generate an iOS build

**Comment by NikolaP93** [comment]
> Using a terminal with Rosetta enabled did worked for me 😉

Praise the lord!

For anyone on macbook m1, this was the only fix that worked

**Comment by Codife** [comment]
>wish I was this lucky lol

**Comment by zahrafalah** [comment]
I fixed it by downgrading my Xcode to 13.2.

**Comment by amirhosein5858** [comment]
maybe this helped someone
i faced with this error and the problem was duplicate resources in `build phases` ->  Copy pods recources
in my case there was many duplicate fonts

**Comment by 6zc** [comment]
For me, it was just a silly mistake. Don't contain any space " " in your path. It seems pod would cut the path at " ".

**Comment by azizmobarak** [comment]
if someone get it check if you have permission if not run with `sudo`

**Comment by vikas6377297826** [comment]
> For me, it was just a silly mistake. Don't contain any space " " in your path. It seems pod would cut the path at " ".

thank you bro i wasted two day of this

**Comment by joaogabrielramos28** [comment]
> For me, it was just a silly mistake. Don't contain any space " " in your path. It seems pod would cut the path at " ".

Thank you so much bro. Solved my problem !

**Comment by ishaqashraf** [comment]
had the same issue. 
what worked for me is to change the `appExample.xcworkspace` to `AppExample.xcworkspace`

**Comment by leonardorb** [comment]
```
cd ios
rm -rf build
rm -ft Pods
pod install
cd ..
npx react-native run-ios
```

**Comment by Kyu** [comment]
Adding my dev account fixed this

**Comment by ConnorDuncan** [comment]
For me I was using React-Native-Firebase and had a duplicate GoogleService-Info.plist file. I opened xcode, deleted it in the directory, and it started working.
